ansible_collections/arista/avd/roles/eos_designs/templates/overlay/ebgp/address-family-ipv6.j2

```diff
@@ -0,0 +1,4 @@
+  address_family_ipv6:
```

Comment:
Is this needed? Are we activating the overlay peer-group on AF ipv6 by default? We _are_ on ipv4 hence the specific deactivation there. If we need it, it should be done on the underlay_ipv4 PR, since that is when the AF is created first.